### DISCUSSION TITLE
Use of DB Table

### DISCUSSION THREAD

**Discussion by pranay-garg-groww** [featurehub-io/featurehub#909] [Q&A]
What is the use of the following tables in DB?
fh_app_strategy
fh_strat_for_feature

I played around by creating applications, features and their strategies and there is no data beign populated in the above tables.

**Comment by rvowles** [comment]
Hey, these are for shared strategies - they are available in the API but not yet in  the application. Plans for these are on the drawing board for the early part of this year.
